src/tests/unit/containers/my-certificates/certificate-item/certificate-item.spec.js

```diff
@@ -0,0 +1,60 @@
+import React from 'react';
+import { screen, render, fireEvent } from '@testing-library/react';
+import { mockedCertificateItem } from './certificate-item.variable';
+import CertificateItem from '../../../../../containers/my-certificates/cetrificate-item';
+
+jest.mock(
+  '../../../../../containers/my-certificates/cetrificate-item/certificate-item.styles.js',
+  () => ({
+    useStyles: () => ({})
+  })
+);
+
+jest.mock('@material-ui/styles', () => ({
+  ...jest.requireActual('@material-ui/styles'),
+  useTheme: () => ({
+    palette: {
+      type: 'light'
+    }
+  })
+}));
+
+jest.mock('react-i18next', () => ({
```

Comment:
Doest we have a global mock for i18next? if not add one.

Reply:
You are right, there is a global mock. I deleted  it